README.md

```diff
@@ -107,6 +107,26 @@ get({
 })
 ```
 
+### JSON Handling
+You can serialize/deserialize request and response with JSON:
```

Comment:
Can you add a newline before and after this line so it matches the rest of the readme?